src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/crushmap/crushmap.component.spec.ts

```diff
@@ -64,24 +64,88 @@ describe('CrushmapComponent', () => {
           { children: [1, 0, 2], type: 'host', name: 'my-host', id: -2 },
           { status: 'up', type: 'osd', name: 'osd.0', id: 0 },
           { status: 'down', type: 'osd', name: 'osd.1', id: 1 },
-          { status: 'up', type: 'osd', name: 'osd.2', id: 2 }
+          { status: 'up', type: 'osd', name: 'osd.2', id: 2 },
+          { children: [-4], type: 'root', name: 'default-2', id: -3 },
+          { children: [4], type: 'host', name: 'my-host-2', id: -4 },
+          { status: 'up', type: 'osd', name: 'osd.0-2', id: 4 }
         ]);
       });
 
-      it('should have tree structure derived from a root', () => {
-        expect(component.tree.value).toBe('default (root)');
-      });
-
       it('should have one host child with 3 osd children', () => {
```

Comment:
maybe refine description in `it()` a little bit since now testing 2 roots.

Reply:
Done